### PR TITLE
NGL preliminary documentation

### PR DESCRIPTION
**Pull request by sjbrandenberg**
I put together all of our NGL documentation into a single usecase.md file. It's a lot of content, but I think it works OK as a single file.

### CODE REVIEW

**Review by kks32 — APPROVED** [review]
Looks good to me.

You can choose to split into multiple files for installation, set-up, etc...if you like. As long as we point the root to usecase.md so we are consistent.

File: docs/brandenberg/usecase.md [hunk 1]
```diff
@@ -1,89 +1,893 @@
...
+users can interact with it using Jupyter notebooks.
 
-* Please cite [AUTHORS et al. (20xx) - example of published project](https://doi.org/10.17603/ds2-3zdj-493) to acknowledge the use of any resources from this use case.
+## Citations and Licensing
+
+* Please cite [Zimmaro, P., et al. (2019)](https://doi.org/10.21222/C2J040) to acknowledge the use of the NGL Database. Data in the NGL database has been gathered from [these](https://nextgenerationliquefaction.org/citations.php) published sources. If you use specific data in the database, please cite the original source.
 
 * Please cite [Rathje et al. (2017)](https://doi.org/10.1061/(ASCE)NH.1527-6996.0000246) to acknowledge the use of DesignSafe resources.  
 
 * This software is distributed under the GNU General Public License (https://www.gnu.org/licenses/gpl-3.0.html).
```
Comment (kks32) [inline docs/brandenberg/usecase.md]:
```suggestion
* This software is distributed under the [GNU General Public License](https://www.gnu.org/licenses/gpl-3.0.html).  
```

File: docs/brandenberg/usecase.md [hunk 2]
```diff
@@ -1,89 +1,893 @@
...
-### Description 
+## Understanding the Database Schema
+
+The NGL database is organized into tables that are related to each other via keys. To query the database, 
+you will need to understand the organizational structure of the database, called the schema. The database 
+schema is documented at the following URL:
+
+<https://nextgenerationliquefaction.org/schema/index.html>
+  
+## Installing Database Connection Script
```
Comment (kks32) [inline docs/brandenberg/usecase.md]: Needs discussion on where this code can be run, i.e., introduce Jupyter

File: docs/brandenberg/usecase.md [hunk 3]
```diff
@@ -1,89 +1,893 @@
...
 * This software is distributed under the GNU General Public License (https://www.gnu.org/licenses/gpl-3.0.html).  
 
-### Description 
+## Understanding the Database Schema
+
+The NGL database is organized into tables that are related to each other via keys. To query the database, 
+you will need to understand the organizational structure of the database, called the schema. The database 
+schema is documented at the following URL:
+
+<https://nextgenerationliquefaction.org/schema/index.html>
```
Comment (kks32) [inline docs/brandenberg/usecase.md]:
```suggestion
[https://nextgenerationliquefaction.org/schema/index.html](https://nextgenerationliquefaction.org/schema/index.html)
```

File: docs/brandenberg/usecase.md [hunk 4]
```diff
@@ -1,89 +1,893 @@
...
+![Screenshot of result of query of SITE table data.](img/SiteTableQuery.png)
 
-*Add images to the folder img and use relative path to specify the location of the image.*   
+**Figure 1.** Results of query of SITE table data.
 
-![caption](img/mkdocs-template.png)
-> Use case template design
+### Query Wildlife liquefaction array data
 
+This cell queries event information from the EVNT table and surface evidence of liquefaction information from the FLDM table at the Wildlife Array site. The definition of each table and site is below. The query utilizes an INNER JOIN statement to combine tables based on shared keys, and will return all values that have matching keys in both tables. For more details, see <https://www.w3schools.com/sql/sql_join_inner.asp>
```
Comment (kks32) [inline docs/brandenberg/usecase.md]:
```suggestion
This cell queries event information from the `EVNT` table and surface evidence of liquefaction information from the `FLDM` table at the Wildlife Array site. The definition of each table and site is below. The query utilizes an `INNER JOIN` statement to combine tables based on shared keys, and will return all values that have matching keys in both tables. For more details, see [https://www.w3schools.com/sql/sql_join_inner.asp](https://www.w3schools.com/sql/sql_join_inner.asp)
```

File: docs/brandenberg/usecase.md [hunk 5]
```diff
@@ -1,89 +1,893 @@
...
+command = 'SELECT (SELECT COUNT(SCPG_ID) FROM SCPG) as "CPT Soundings", '
+command += '(SELECT COUNT(BORH_ID) FROM BORH) as "Boreholes", '
+command += '(SELECT COUNT(GSWG_ID) FROM GSWG) as "Surface Wave Measurements", '
+command += '(SELECT COUNT(GINV_ID) FROM GINV) as "Invasive VS Measurements", '
+command += '(SELECT COUNT(FLDM_ID) FROM FLDM WHERE FLDM_SFEV=1) as "Liquefaction Observations", '
+command += '(SELECT COUNT(FLDM_ID) FROM FLDM WHERE FLDM_SFEV=0) as "Non-Liquefaction Observations"'
+
+scpg_count = ngl.read_sql(command).T
+scpg_count.columns=["Count"]
+scpg_count
```
Comment (kks32) [inline docs/brandenberg/usecase.md]: Add a code section using ```

**Review by kks32 — CHANGES_REQUESTED** [review]
Please add a ## Background section at the start and other suggested changes

File: docs/brandenberg/usecase.md (hunk 2 above)
Comment (sjbrandenberg) [reply docs/brandenberg/usecase.md]: I added a section called "Querying Data via Jupyter Notebooks" that provides this context.

File: docs/brandenberg/usecase.md (hunk 5 above)
Comment (sjbrandenberg) [reply docs/brandenberg/usecase.md]: I have removed all code blocks from the documentation

**Review by kks32 — APPROVED** [review]

**Pull request merged**